### ISSUE TITLE
mathjax渲染问题

### ISSUE THREAD

**Issue by lushuangning** [shd101wyy/markdown-preview-enhanced#1183]
我把渲染引擎改成了`mathjax`，但是在对公式编号和引用的时候，渲染上出现了错误，举例如下。

```
在公式$\eqref{1}$中, 我们发现了一个有趣的现象

$$
\begin{equation}
  \int_0^\infty \frac{x^3}{e^x-1}\,dx = \frac{\pi^4}{15}
  \label{1}
\end{equation}
$$
```

插件渲染出来的效果如图  

![20190808233247](https://user-images.githubusercontent.com/17777259/62716782-4bd82400-ba35-11e9-8596-a564a5a11d23.png)

但是右键选择在浏览器中打开，显示却是正常的  

![20190808233457](https://user-images.githubusercontent.com/17777259/62716740-3bc04480-ba35-11e9-8ef0-3f3ace4421ea.png)

只要添加`\label{}`标签，渲染就会出问题，因此我查阅了`MathJax`官方文档的[公式编号](https://docs.mathjax.org/en/latest/tex.html#automatic-equation-numbering)部分，确认我的写法是正确的。

不知是我设置的问题还是插件渲染的问题。

我的`markdown-preview-enhanced`版本为`0.16.3`，`atom`版本为`1.39.1`

谢过作者

**Comment by hjnnjh** [comment]
同样的问题

**Comment by tmori3y2** [comment]
Use \tag instead of \label.

**Comment by tmori3y2** [comment]
https://www.mathjax.org/#demo

No error, but \label is not interpreted by mathjax.
